### DISCUSSION TITLE
Doris无法使用Navicat Premium 16版本进行连接

### DISCUSSION THREAD

**Discussion by dev-tonic** [apache/doris#17758] [A - General / Q&A]
我在部署完成Doris之后，使用IntelliJ IDEA连接Doris可以连接上，使用TablePlus也能够连接的上，但是使用Navicat Premium16版本的进行连接之后会爆出一个错误。我在进行Test Connection的时候可以连接通过，但是当我点击OK后sava保存之后在双击那个连接，便会报出一个`1105 - errCode = 2, detailMessage = There is no scanNode Backend available.[10002: not alive]`，因此我不知道是我的配置操作出现了问题还是说Doris并不支持Navicat Premium16系统，但如果不支持的话为什么Test Connection的时候能够通过，希望能够得到社区的回复。
<img width="672" alt="CleanShot 2023-03-14 at 09 20 30@2x" src="https://user-images.githubusercontent.com/75539269/224867796-02ab72ad-e140-4a65-a187-0cbd2eafdff3.png">

<img width="369" alt="CleanShot 2023-03-14 at 09 23 34@2x" src="https://user-images.githubusercontent.com/75539269/224868123-332215c7-ae44-484c-8bcc-ed5a3da8ebf1.png">

**Comment by LemonLiTree** [comment]
出现这个报错的时候，show backends\G 看下be是不是都alive

**Reply by dev-tonic** [reply]
<img width="767" alt="CleanShot 2023-03-14 at 11 14 57@2x" src="https://user-images.githubusercontent.com/75539269/224883948-102a7f11-6805-413d-857d-5fbee3bfcb20.png">

**Reply by dev-tonic** [reply]
我去查看了fe.log的日志发现了一个org.apache.thrift.transport.TTransportException: Invalid port -1

**Reply by LemonLiTree** [reply]
be 挂了
